src/pages/FavoriteQueries/FavoriteQueries.ts

```diff
@@ -0,0 +1,80 @@
+import * as vscode from 'vscode';
+import * as path from 'path';
+import { logger } from '../../logger/logger';
+import { showFavoriteQueries } from '../../webViews/favoriteQueries.webiew';
+import { applyQuery } from '../../commands/queryHistory/applyQuery';
+import { Memory, getUUID } from '../../util/util';
+import { deleteFavoriteQuery } from '../../util/favoriteQuery';
+import { IConnection } from '../../types/IConnection';
+
+export interface IFavoriteQueriesWebviewParams {
+    styleSrc: vscode.Uri;
+}
+
+export interface IFavoriteQueriesWebviewState {
+    webviewPanel: vscode.WebviewPanel
+}
+
+export const fetchFavoriteQueries = (context: vscode.ExtensionContext) => {
+    const connection = Memory.state.get<IConnection>("activeConnection");
+    if (!connection) {
```

Comment:
```
    const connection = Memory.state.get<IConnection>("activeConnection");
    if (!connection) {
        vscode.window.showErrorMessage("Please connect to a cluster before opening favorite queries");
        return;
    }
```
    
   this code is not needed here. We don't need to check if connected for fav query

Reply:
I agree, but when we try to paste it fails
Maybe I can add this part to paste functionality only.
Eill update

Reply:
Removed from here